x-pack/plugins/fleet/server/services/elastic_agent_manifest.ts

```diff
@@ -79,7 +79,7 @@ spec:
               # If you are not using this integration, then these capabilites can be removed.
                 - BPF # (since Linux 5.8) allows loading of BPF programs, create most map types, load BTF, iterate programs and maps.
                 - PERFMON # (since Linux 5.8) allows attaching of BPF programs used for performance metrics and observability operations.
-                - SYS_RESOURCE # Allow use of special resources or raising of resource limits. Used by Defend for Containers to modify rlimit_memlock
+                - SYS_RESOURCE # Allow use of special resources or raising of resource limits. Used by `Defend for Containers` to modify `rlimit_memlock`
```

Comment:
This PR is on the way to fix it
(Literally speaking many teams are changing our manifests, eg here came from this [PR](https://github.com/elastic/elastic-agent/pull/2283))

Nevertheless if we see that this happens too often we will need to automate the test

Reply:
@hop-dev I know that we are the codeowners of that manifest but the changes has been made by the security team. Looking at the git history on that file, I can see those changes were introduced 2 weeks ago by @norrietaylor .

Reply:
I just wanted to point out (in case it was necessary) that I didn't want to blame anyone. I wrote my comment to raise visibility in the security team about this issue